client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -79,11 +83,12 @@ const StockListComponent: FC = () => {
         },
       ],
       ['locationName', { sortable: false }],
-      [
-        'itemUnit',
-        { accessor: ({ rowData }) => rowData.item.unitName, sortable: false },
-      ],
-      'packSize',
+      {
+        key: 'packUnit',
+        label: 'label.pack-unit',
+        Cell: getItemVariantPackUnit(rowData => rowData.itemId),
+        accessor: ({ rowData }) => rowData.packSize,
```

Comment:
the double accessor is pretty funky - it is assigned here, then both used and replaced in the Cell Element. The dependency would be good to have commented somewhere - perhaps in the ItemVariantPackUnit to say that the accessor needs to pass in the pack size?

You don't want to have the ItemVariantPackUnit fetch the itemId and packSize directly?

Reply:
I like that ! I had to abstract InnerBasicCell though, is that ok ? 
And is it ok that accessor and formatter is ignored for this type of cell ?
7fcec00062b959bea245097f4bf1dacddf1c9834

Reply:
very neatly done! that looks good - I think it's ok to have the accessor and formatter ignored 🤷 It doesn't feel great going against the pattern.. but the result is clear and easy to use.